source/content/sso-organizations.md

```diff
@@ -6,7 +6,7 @@ tags: [security, organizations]
 ---
 Single sign-on (SSO) allows users to authenticate against your Identity Provider (IdP) when logging into the Pantheon Dashboard. For more information, see [SSO and Identity Federation on Pantheon](/sso).
 
-SAML SSO is included for customers with Diamond Support, and is available for purchase for Pantheon Organizations with Platinum or below. SAML is an organization-wide feature only and is not available on a per site or per environment basis. If you'd like to upgrade to an eligible support package, please contact [Sales](https://pantheon.io/plans/elite?docs). Agencies interested in SAML SSO should reach out to their Partner Manager to see if they qualify. You must be part of the [Pantheon Partner Program](https://pantheon.io/plans/partner-program?docs) to qualify.
+SAML SSO is included for customers with Diamond Support, and is available for purchase for Pantheon Organizations with Platinum or below. Once enabled, SAML applies to an entire email domain, or multiple email domains, and is not available on a per site, per environment, or per user basis. If you'd like to upgrade to an eligible support package, please contact [Sales](https://pantheon.io/plans/elite?docs). Agencies interested in SAML SSO should reach out to their Partner Manager to see if they qualify. You must be part of the [Pantheon Partner Program](https://pantheon.io/plans/partner-program?docs) to qualify.
```

Comment:
@allenfear should we also add a note to say that "available for purchase for Pantheon Organizations with Platinum or below" applies to Enterprise? Looks like it's different for EDUs and Partners

Reply:
Good catch. I'll edit that so we don't get too lost in the nuances as to distract from the main point.